src/payment/PaymentForm.jsx

```diff
@@ -104,6 +105,21 @@ export class PaymentFormComponent extends React.Component {
     return requiredFields;
   }
 
+  handleTrackEvent() {
```

Comment:
I would change this name to something like `handleSubmitButtonClick` or something. Probably better to describe the event it handles rather than what it will do when that even happens.